style/Badges.css

```diff
@@ -0,0 +1,78 @@
+.badge-items {
+  align-items: flex-end;
+  justify-content: center;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  background-color: transparent;
+  border: 0.5px solid rgba(128, 128, 128, 0.363);
+  margin: 10px;
+}
+.badge-button {
+  position: relative;
+  margin: 10px;
+  color: var(--black-color);
+  background-color: transparent;
+  border: none;
+  outline: none;
+  border-radius: 50%;
+}
+.badge-items .kitty {
+  padding: 10px;
+}
+.badge-avatar {
+  position: relative;
+  align-items: flex-end;
+  justify-content: center;
+  margin: 25px;
```

Comment:
remove margins for components, so that component can be used without worrying about why it is not positioning as per my requirement

Reply:
okay